pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix

```diff
@@ -5,6 +5,8 @@ let
   inherit (self) callPackage;
 in
 {
+  inherit (pkgs) emacspeak;
```

Comment:
Applying this patch can fix issues mentioned [here].  However, whether this package should be moved to "by-name" needs discussion.

```patch
-   inherit (pkgs) emacspeak;
+   emacspeak = pkgs.emacspeak.override { emacs = self.emacs; };
```

[here]: https://github.com/NixOS/nixpkgs/pull/291953/files#r1795973277